docs/how-to/managed-data/fetch-data.md

```diff
@@ -0,0 +1,54 @@
+
+
+
+## Fetch databases 
+
+
+=== "CLI"
+
+    Databases can be viewed in the command line using:
+
+    ```bash
+    deeporigin data show-db <id-of-database>
+    ```
+
+    The CLI can also return JSON output that can be piped
+    to a JSON parser:
+
+    ```bash
+    deeporigin data show-db <id-of-database> --json | jq
+    ```
+
+    This is useful to quickly view the data in the database. 
+    To save this database to a CSV, use:
+
+    ```bash
+    deeporigin data cp deeporigin://<id-of-database> <local-folder>
+    ```
+
+    The `deeporigin://` prefix identifies a resource as
+    existing on Deep Origin, and the `cp` command downloads 
+    the database as a CSV and all included files are downloaded 
+    to the `<local-folder>`
```

Comment:
Add a period at the end of the paragraph.

Reply:
fixed